### PR TITLE
Add storyboard for UoN, IPSL and KNMI

### PR DESCRIPTION
**Pull request by geek-yang**
- Upload presentations to the EUCP folder on sharepoint
- Convert powerpoints from UoN, IPSL and KNMI to stories
- Storyboards added to the storyboard viewer and accessible online

### CODE REVIEW

File: content/carribean/1_intro.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+image: carribean/carribean.png
```
Comment (Peter9192) [inline content/carribean/1_intro.md:2]: I think we can probably combine the first and second panel.

File: content/carribean/3_TCvariability.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+image: carribean/TCvariability.png
+headline: TC variability
+---
+
+## TC variability is huge
```
Comment (Peter9192) [inline content/carribean/3_TCvariability.md:6]: TC = Tropical Cyclone. Let's write it out full, both in the title and the headline.

File: content/rainstorms/2_epp.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+image: rainstorms/epp_sepp.png
+headline: EPP and SEPP
```
Comment (Peter9192) [inline content/rainstorms/2_epp.md:3]:
```suggestion
image: rainstorms/epp_sepp.png
headline: Precipitation metrics
```

File: content/rainstorms/8_conclusion.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
...ecipitation. Our results suggest that storms will have higher peak intensity, longer duration and will be more frequent across the whole of Europe. Current storms already produce a large number of flash floods, with their potential impact depending on land use, terrain slope, drainage, and other factors. SEPP increases would significantly increase this flash flood potential, as an MCS would be more likely to “stagnate” on a locality, exposing it to extreme precipitation of longer duration. Additionally, storm system movement is associated with upper level winds and, hence, large-scale dynamics. This may increase fluvial flood risk through consecutive events in one favorable synoptic setting spanning days or more; blocking is also favored by large-scale meandering patterns associated with slower flows. Understanding the underlying ingredients for heavy precipitation change is crucial from an impacts perspective, helping to discriminate controlling factors, which have wider applicability beyond those for a single accumulation period, and to identify the reliability of projected changes. This suggests that future studies should focus on precipitation accumulations over space and time.
```
Comment (Peter9192) [inline content/rainstorms/8_conclusion.md:8]: Maybe use some paragraphs here?

**Review by Peter9192 — APPROVED** [review]
Nice, everything works and I'll merge it right away as we want to iterate quickly. However, I do have a couple of comments that I'd like to address in follow-ups. In addition to the specific ones:

- Let's try to have short paragraphs.
- Don't overuse abbreviations, and if used, make sure they are explained
- We can ask for better quality figures when deemed necessary
- Take the liberty to adjust the visuals, rewrite the text, or combined multiple panels into one. We don't have to be indifferent about the content; we have relevant expertise both on content and on science communication. If we see opportunities to strengthen the storyboard, let's implement them.

File: content/carribean/1_intro.md (hunk 1 above)
Comment (geek-yang) [reply content/carribean/1_intro.md:2]: Good suggestion! They are combined now!

File: content/carribean/3_TCvariability.md (hunk 2 above)
Comment (geek-yang) [reply content/carribean/3_TCvariability.md:6]: Corrected.

File: content/rainstorms/2_epp.md (hunk 3 above)
Comment (geek-yang) [reply content/rainstorms/2_epp.md:3]: Thanks for the comment. It is changed now.

File: content/rainstorms/8_conclusion.md (hunk 4 above)
Comment (geek-yang) [reply content/rainstorms/8_conclusion.md:8]: Good point. The content is now split into three paragraphs.

**Pull request merged**